src/components/Navbar/NavbarLinkMobile.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useLocation, Link } from 'react-router-dom';
+import { Button, MenuItem, Text } from '@chakra-ui/react';
+
+const NavbarLinkMobile = ({ text, path = '/' }) => {
+  const current = useLocation().pathname;
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'current' is assigned a value but never used.